src/Lambda/LambdaClient.php

```diff
@@ -46,8 +46,6 @@
  * @method \GuzzleHttp\Promise\Promise getPolicyAsync(array $args = [])
  * @method \Aws\Result invoke(array $args = [])
  * @method \GuzzleHttp\Promise\Promise invokeAsync(array $args = [])
- * @method \Aws\Result invokeAsync(array $args = [])
- * @method \GuzzleHttp\Promise\Promise invokeAsyncAsync(array $args = [])
```

Comment:
Oh I not see invokeAsync in api-2.json sorry -_-'
Yeah ok no problem.

ps: This is not strange to Async an Async function ?

Reply:
`invokeAsync` is a Lambda service operation. The PHP SDK also uses `Async` as a suffix to indicate that a promise will be returned (instead of a result object). So, `invokeAsyncAsync` is indeed valid for the SDK, strangely enough! The issue we will have to address internally is how to define the non `Async` version of functions that actually end in the word `Async`.

Reply:
Now fix normally